drivers/bluetooth/hci/rpmsg.c

```diff
@@ -45,6 +45,17 @@ static bool is_hci_event_discardable(const uint8_t *evt_data)
 		switch (subevt_type) {
 		case BT_HCI_EVT_LE_ADVERTISING_REPORT:
 			return true;
+#if defined(CONFIG_BT_EXT_ADV)
+		case BT_HCI_EVT_LE_EXT_ADVERTISING_REPORT:
+		{
+			const struct bt_hci_evt_le_ext_advertising_report *ext_adv =
+				(void *)&hci_buf[3];
+
+			return (ext_adv->num_reports == 1) &&
+				   ((ext_adv->adv_info->evt_type &
```

Comment:
Minor, but would make it more clear that we are only checking the first report:
```suggestion
				   ((ext_adv->adv_info[0].evt_type &
```

Reply:
fixed this one too.